### PR TITLE
Jolo

### PR DESCRIPTION
**Pull request by maxsumrall**

### CODE REVIEW

File: _posts/2019-08-06-Jolo.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: Jolo
```
Comment (Stephan202) [inline _posts/2019-08-06-Jolo.md]:
```suggestion
title: jOLo
```

File: _posts/2019-08-06-Jolo.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+technology: Java
+image: jolo.jpg
+desc: A jOOQ utility library for object loading
+proj-url: https://www.github.com/picnicsupermarket/jolo
+proj-num: 07
```
Comment (Stephan202) [inline _posts/2019-08-06-Jolo.md]: Octal? :thinking:

File: _site/feed.xml [hunk 3]
```diff
@@ -3,79 +3,79 @@
...
     <title>picnic.tech</title>
     <description>A humble milkman standing on the shoulder of giants</description>
-    <link>http://localhost:4000</link>
-    <atom:link href="http://localhost:4000/feed.xml" rel="self" type="application/rss+xml" />
+    <link>http://picnic.tech/</link>
+    <atom:link href="http://picnic.tech//feed.xml" rel="self" type="application/rss+xml" />
     
       <item>
-        <title>Reactive Support</title>
+        <title>Jolo</title>
```
Comment (Stephan202) [inline _site/feed.xml]:
```suggestion
        <title>jOLo</title>
```

File: _site/feed.xml [hunk 4]
```diff
@@ -3,79 +3,79 @@
...
         <description>
 </description>
-        <pubDate>Fri, 05 Jan 2018 00:00:00 +0100</pubDate>
-        <link>http://localhost:4000/Reactive-Support/</link>
-        <guid isPermaLink="true">http://localhost:4000/Reactive-Support/</guid>
+        <pubDate>Tue, 06 Aug 2019 00:00:00 -0500</pubDate>
+        <link>http://picnic.tech//Jolo/</link>
+        <guid isPermaLink="true">http://picnic.tech//Jolo/</guid>
```
Comment (Stephan202) [inline _site/feed.xml]: ^ Two times: double slash? :thinking: 

(Hah, also below.)

File: _posts/2019-08-06-Jolo.md (hunk 2 above)
Comment (maxsumrall) [reply _posts/2019-08-06-Jolo.md]: Come again?

File: _posts/2019-08-06-Jolo.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: Jolo
+technology: Java
+image: jolo.jpg
+desc: A jOOQ utility library for object loading
```
Comment (Stephan202) [inline _posts/2019-08-06-Jolo.md]: Shouldn't we use the same description as on Github?

File: _posts/2019-08-06-Jolo.md (hunk 2 above)
Comment (Stephan202) [reply _posts/2019-08-06-Jolo.md]: Haha, I mean, "do we need that leading zero?"

Comment (maxsumrall) [reply _posts/2019-08-06-Jolo.md]: Ah, it's used in the other posts so I assume its a jekyll thing

Comment (Stephan202) [reply _posts/2019-08-06-Jolo.md]: Just tried: if I remove them then that has no impact on the generated code. So push a commit to drop them.

**Pull request merged**